Intro to open source with GitHub/README.md

```diff
@@ -0,0 +1,22 @@
+# How to work on opensource contribution with Git
+
+## Making a contribution
+1. Fork the repo which you want to work on (eg. https://github.com/take2rohit/sih_number_plate is original repo where you want to contribute) 
+1. Clone the forked repository (not the above repo link)
```

Comment:
numbering is 1, should be 2 (but it still renders correctly, but just change it 😂 )

Reply:
Add some clarity by saying 'Clone the forked repository from your own profile (NOT the original repository)'

Reply:
numbering doesnt matters!